### PR TITLE
docs: Correct assumption of global CSS reset

### PR DESCRIPTION
**Pull request by lukelooker**
Minor corrects to documentation site styling due to refactor of @looker/components CSS reset removal.

### CODE REVIEW

File: www/src/components/HeaderContent/AppLogo.tsx [hunk 1]
```diff
@@ -46,6 +46,7 @@ export const AppLogo = styled(AppLogoLayout)`
   align-items: center;
   display: flex;
   height: 1.5rem;
+  text-decoration: none;
```
Comment (lukelooker) [inline www/src/components/HeaderContent/AppLogo.tsx:49]: We no longer style all `a`. Would love to come up with a good pattern to combine a third-party `Link` (Gatsby or React Router) with our `Link` to get functionality + styling.

File: www/src/Layout/Layout.tsx [hunk 2]
```diff
@@ -77,7 +78,7 @@ export const Layout: FC = ({ children }) => {
     <ComponentsProvider loadGoogleFonts themeCustomizations={customTheme}>
       <GatsbyOverrides />
       <MDXProvider components={MDXComponents}>
-        <Page>
+        <Page fixed>
```
Comment (lukelooker) [inline www/src/Layout/Layout.tsx:81]: Specifies `height="100%"`

File: www/src/components/HeaderContent/HeaderContent.tsx [hunk 3]
```diff
@@ -124,6 +124,7 @@ export const NavigationList = styled.nav`
   ul,
   li {
     height: 100%;
+    list-style: none;
```
Comment (lukelooker) [inline www/src/components/HeaderContent/HeaderContent.tsx:127]: I suppose we should eventually build a component for this kind of horizontal nav group. 

@jhardy Food for thought in list/nav 🤔

File: www/src/components/Navigation/Section.tsx [hunk 4]
```diff
@@ -77,4 +77,5 @@ export const Section: FC<SectionProps> = ({ section }) => {
 const PageList = styled.ul`
   font-size: ${({ theme }) => theme.fontSizes.small};
   list-style-type: none;
+  padding: 0;
```
Comment (lukelooker) [inline www/src/components/Navigation/Section.tsx:80]: This should probably switch to being one of our `List` components / compositions eventually.

**Review by mdodgelooker — APPROVED** [review]
LGTM!

**Pull request merged**